src/models/ActionItem.js

```diff
@@ -59,6 +59,7 @@ module.exports = function(formio) {
 
   try {
     model.schema.index({created: 1}, {expireAfterSeconds: 2592000});
+    model.schema.index({_ts: 1}, {expireAfterSeconds: 2592000});
```

Comment:
These should be in different try/catch statements. If they are on Azure, it will crash on the first statement and not make it to the second, and if you reverse the order to fix that problem, the other will occur on non-azure platforms.